### PR TITLE
replacing lerna --parallel by concurrently like in sdk

### PR DESCRIPTION
**Pull request by acourdavault**
lerna not needed anymore
## Description
removed lerna from package.json scripts
ran `yarn`

No new warning in yarn.

Cost of modules install:
- before 155
- after 108 MB

Yarn lock -1500 lines

You can see the changes are ONLY removing dependancies

There is no change in any of the resolved packages (only removals)

## Type of Change
- Non breaking change ('update', 'change')

## Applicable E2E Tests
should not impact production but we must test all of it

### CODE REVIEW

**Comment by acourdavault** [comment]
**WAIT for PR 44 on groovy build script (jenkins)**

File: yarn.lock [hunk 1]
```diff
@@ -1441,80 +1441,6 @@
   resolved "https://registry.yarnpkg.com/@emotion/weak-memoize/-/weak-memoize-0.2.4.tgz#622a72bebd1e3f48d921563b4b60a762295a81fc"
   integrity sha512-6PYY5DVdAY1ifaQW6XYTnOMihmBVT27elqSjEoodchsGjzYlEsTQMcEhSud99kVawatyTZRTiVkJ/c6lwbQ7nA==
```
[the remark this answers is not in the document]
Comment (acourdavault) [reply yarn.lock]: Removed lerna json

**Comment by acourdavault** [comment]
@jstamis-linode I just cleaned up the doc,

Note:
- Releasing, commands use Yarn instead of lerna
- start_manager.sh : it's updated, but do we use this script?
- Dockerfile: updated too, but i m not sure if this file is still useful

**Comment by jstamis-linode** [comment]
> @jstamis-linode I just cleaned up the doc,
> 
> Note:
> 
> * Releasing, commands use Yarn instead of lerna
> * start_manager.sh : it's updated, but do we use this script?
> * Dockerfile: updated too, but i m not sure if this file is still useful

Not sure how we will use them or not at the moment.

**Pull request merged**